libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -29,28 +29,24 @@ const val DEFAULT_NAVIGATOR_PREDICTION_MILLIS = 1100L
  * @param isFromNavigationUi Boolean *true* if is called from UI, otherwise *false*
  * @param isDebugLoggingEnabled Boolean
  * @param deviceProfile [DeviceProfile] defines how navigation data should be interpretation
+ * @param builder [Builder] used the create the [NavigationOptions]
  */
-data class NavigationOptions constructor(
+data class NavigationOptions(
     val accessToken: String?,
     @TimeFormat.Type val timeFormatType: Int,
     val navigatorPredictionMillis: Long,
     val distanceFormatter: DistanceFormatter?,
     val onboardRouterConfig: MapboxOnboardRouterConfig?,
     val isFromNavigationUi: Boolean = false,
     val isDebugLoggingEnabled: Boolean = false,
-    val deviceProfile: DeviceProfile
+    val deviceProfile: DeviceProfile,
+    val builder: Builder
```

Comment:
This makes it a SEMVER. But this will also make it obvious to anyone who is constructing the NavigationOptions without the builder, will need to use the Builder